src/validation.h

```diff
@@ -564,6 +564,8 @@ class CChainState
     //! mempool that is kept in sync with the chain
     CTxMemPool& m_mempool;
 
+    const CChainParams& m_params;
```

Comment:
nit: This member variable looks like the only one without a Doxygen comment in this section. Maybe add one?

Is `protected` better here than `private`? It compiled for me even when the variable is `private`.

Reply:
Thanks, fixed. Also fixed up some typos.

Reply:
Actually, changing to `private` didn't compile, so I left this as-is for now.

Reply:
I moved only `const CChainParams& m_params;` to `private` section and it did compile for me. It confuses me but ok.

Reply:
`private` and `protected` are the same anyway, unless the class is derived